.gitignore

```diff
@@ -49,3 +49,5 @@ docs/source/adaptors/saga.adaptor*
 
 # PyCharm
 .idea
+/.pycharmrc
+Supprimerait .pycharmrc
```

Comment:
Sorry for this file, I omit to add it to gitignore, 'supprimerait' means 'may remove' (I guess if present)